spec/poros/quote_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Quote do
+  before :each do 
+    @quote = QuotesFacade.search_by_keyword("I am").first
```

Comment:
This definitely works, but there are a couple things I don't love about this approach. 1) If your facade ever change (ex: the name of the method), then this test breaks even though it's unrelated. 2) It's making an API call, when all we really need to test this is some sample data in the format that the `Quote` class is expecting. We don't need to use VCR or other mocking tools to do this. 3) If the first result from the API ever changes, this test fails. We can just make up data ourselves and do something like this instead:
```ruby
  before :each do
    @quote = Quote.new({ content: "If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?",
                         author: "Rabbi Hillel",
                         tags: ["famous-quotes", "wisdom"] })
  end
```